python/prairielearn.py

```diff
@@ -75,6 +77,10 @@ def to_json(v):
     """
     if np.isscalar(v) and np.iscomplexobj(v):
         return {'_type': 'complex', '_value': {'real': v.real, 'imag': v.imag}}
+    elif np.issubdtype(type(v), np.integer) and type(v).__module__ == 'numpy':
+        return {'_type': 'np_integer', '_concrete_type': type(v).__name__, '_value': int(v)}
```

Comment:
Are we doing something sensible if a number can't be represented as a 32-bit integer? IIRC Numpy ints are 64-bit but JSON can't represent a full 64-bit int.

Reply:
Good catch! We can just always use a string to represent the integer value. I'll also add a test for this.